packages/components-react/src/examples/PinCodeExampleControlled.tsx

```diff
@@ -0,0 +1,21 @@
+import { useCallback, useState } from 'react';
+import type { PinCodeUpdateEvent } from '@porsche-design-system/components-react';
+import { PPinCode, PText } from '@porsche-design-system/components-react';
+
+export const PinCodeExampleControlledPage = (): JSX.Element => {
+  const length = 4;
+  const [currentValue, setCurrentValue] = useState<string[]>();
```

Comment:
When controlled, it should have a value other than undefined?
```suggestion
  const [currentValue, setCurrentValue] = useState<string[]>([]);
```